.gitbook/contents/developer/boba-basics/bridge-basics/README.md

```diff
@@ -4,14 +4,14 @@ description: Learn more about the Boba Network Bridges
 
 # Token Bridging
 
-Although Boba Network is an L2 (and therefore fundamentally connected to Ethereum), it's also a separate blockchain system. App developers commonly need to move data and assets between Boba Network and Ethereum. We call the process of moving data and assets between the two networks "bridging".
+Although Boba Network is an L2 (and therefore fundamentally connected to a L1), it's also a separate blockchain system. App developers commonly need to move data and assets between Boba Network and Ethereum or an alt-L1 like BNB Chain. We call the process of moving data and assets between the two networks "bridging".
```

Comment:
Small grammar point here, inconsistency between "an L2" and "a L1".

Reply:
Good catch, fixed